apps/docs/src/content/learn/basics/loading-assets.mdx

```diff
@@ -4,126 +4,35 @@ title: Loading Assets
 order: -500
 ---
 
-A typical Threlte application will make use of textures, models, and other assets. These assets are loaded using the `useLoader` hook. Assets loaded via `useLoader` are automatically cached and will not be loaded or parsed again if they are already in the cache.
+There are a number of different assets you might like to use in your Threlte application. Below are sections to guide you on how to get started with them. Three.js has a [ton of different loaders](https://threejs.org/docs/index.html?q=loader) and Threlte can use all them. You're recommended to use the [useLoader](/docs/reference/core/use-loader) hook to cache assets and for convenience in asynchronous setups. This will help reduces network requests, bandwidth, memory usage and improve the performance of your application.
 
 <Tip type="info">
-  This section assumes you placed your assets in your public folder or in a place in your
-  application where you can import them easily.
+  It is assumed you placed your assets in your public folder or in a place in your application where
```

Comment:
I'd recommend keeping "This section assumes" or "These instructions assumes". "It is assumed" is more ambiguous and I have a harder time understanding what is meant.

Reply:
```suggestion
  This section assumes you placed your assets in your public folder or in a place in your application where
```